Rust/fizz-buzz.rs

```diff
@@ -0,0 +1,19 @@
+fn _check(num: isize, n: isize) -> bool {
```

Comment:
Just change it to

```rs
fn _check(num: isize, n: isize) -> bool {
    num % n == 0
}
```

The `if` keyword there will generate syntax error. Try to test the code before you create a pr.

Reply:
wait haha lol.. sorry since i just need the PR i do something bad without testing the code